pages/common/git-cp.md

```diff
@@ -0,0 +1,13 @@
+# git cp
+
+> Copy an existing file to a new location, preserving history.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-cp>.
+
+- Copy an existing file in a Git repo, in the same directory:
+
+`git cp {{file}} {{new_file}}`
+
+- Copy an existing file in a Git repo and move it elsewhere:
+
+`git cp {{path/to/file}} {{/directory/to/new_file}}`
```

Comment:
Why did you put a trailing slash here? Also, "directory to new file" is bad grammar ;)
```suggestion
`git cp {{path/to/file}} {{path/to/new_file}}`
```